experience.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<title>Experience</title>
+</head>
+<body>
+	<h1>Jean Michel Experience</h1>
+
+	 <nav>
+      <a href="index.html">About</a>
+      <a href="experience.html">Experience</a>
+    </nav>
+
+    <hr>
+    
+	 <img src="jeancv.jpg">
```

Comment:
:hammer: Is your image showing up? Your `src` attribute is **j**eancv.jpg but your actual filename is **J**eancv.jpg. It's the little things in life :)